tests/README.md

```diff
@@ -27,64 +27,65 @@ accessible via ssh on the libvirt domain and have Ansible installed on the host
 
 *NOTE*: Create snapshot after all these steps, to manually revert in case the
 test suite breaks something and fails to revert. Do not use snapshot names
-starting with `ssg\_`
+starting with `ssg_`
 
-## Two modes of operation
-SSG Test Suite currently supports two ways of DataStream evaluation. Simpler
-one to use is `profile`, which scans and remedies target domain based on
-particular profile.
+## Executing the test suite
 
-Second, more complex, is `rule`, which runs set of validation scenarios
-for particular subset of rules. Each rule has its own scenarios, and each
-scenario is run separately to eliminate need for cleanup.
+### Common options
 
-## Running test suite in profile mode
-Example of evaluation of default profile (common):
+- `--hypervisor`: Typically, you will use the `qemu:///system` value.
+- `--domain`: `libvirt` domain, which is basically name of the virtual machine.
+- `--datastream`: Path to the datastream that you want to use for scanning.
+  It will be transferred to the scanned VM via SSH.
+- `--xccdf-id`: Also known as `Ref-ID`, it is the identifier of benchmark within the datastream.
+  It can be obtained by running `oscap info` over the datastream XML and searching for `Ref-ID` strings.
+- `--help`: This will get you the invocation help.
+
+### Profile-based testing
+In this operation mode, you specify the `profile` command and you supply the profile ID as a positional argument.
```

Comment:
> `profile` command

The `profile` command is the example command below?

Reply:
Yes, that argument is commonly referred to as command or subcommand (e.g. `commit` command of `git`). I agree that subcommand could be easier to understand.